V1.5.1

jeremygurr opened the pull request.

jeremygurr committed:
don't give jiyva at ecumenical altar if player xl is < 15, unless they are on nightmare difficulty

jeremygurr committed:
merge sp into essence for Djinni

jeremygurr committed:
silence automagic low magic warning.

jeremygurr committed:
centralize hp, mp, sp fetching code. And merge sp into Djinni essence.

jeremygurr committed:
airstrike no longer does additional damage to flying players

jeremygurr committed:
increase sif piety gain

jeremygurr committed:
implement instakill protection

see changelog-ca for details

jeremygurr committed:
Merge branch 'master' into v1.5.1

yrmvgh commented:
Re: sif

yrmvgh commented:
if your target is to make channeling usable more frequently

yrmvgh commented:
consider that changing piety gain also affects book gift rate... so just be sure this is the knob you wanna twist!

jeremygurr commented:
Good point, I should reduce book gift rate too.

jeremygurr committed:
reduce sif book gift frequency, and make them better

jeremygurr committed:
lua script bug fix

jeremygurr committed:
Merge branch 'master' into v1.5.1

jeremygurr committed:
accelerate sif piety gain

jeremygurr committed:
ammo removal 1

jeremygurr committed:
Revert "Remove launcher ammo brands"

This reverts commit dbab4ee685ab4b05a1684d4e1dcd9cb8ddeb1a1a.

jeremygurr committed:
ammo removal 2

jeremygurr committed:
polish off ammo removal

jeremygurr committed:
fix dwarf spawning bug

jeremygurr committed:
decouple difficulty modes from piety gain, stat recovery, etc.

before, slower experience gain from nightmare mode also meant slower
piety gain, which threw off all kinds of balance.

jeremygurr committed:
boost fighting and spellcasting aptitudes for species that have particularly low values, since they are so much more important now

jeremygurr committed:
adjusted spirit shield behavior

see changelog-ca

jeremygurr committed:
sif channeling now costs 25 piety instead of 50

jeremygurr committed:
minimum range of flame tongue increased to 2

jeremygurr committed:
summons cap of 5, and spell cost minimum of 5 mp

jeremygurr committed:
bug fixes

jeremygurr committed:
adjust cast failure

jeremygurr committed:
tweak spell failure balance
